packages/public/common-ui-web/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useRef } from 'react';
+import { useInteractiveStatus } from '@monkvision/common';
+import { SliderProps, useSliderStyle } from './useSliderStyle';
+import { useSlider } from './useSlider';
+
+/**
+ * Slider component allows users to select a value within a specified range by dragging a thumb along a horizontal track.
+ */
+export function Slider({
+  min = 0,
+  max = 100,
+  value = (max - min) / 2,
+  primaryColor = 'secondary-xlight',
```

Comment:
For me :
- primaryColor = knob color
- secondaryColor = selected bar color
- tertiaryColor = NOT selected bar color